go/vt/wrangler/reparent.go

```diff
@@ -1085,3 +1088,23 @@ func (wr *Wrangler) TabletExternallyReparented(ctx context.Context, newMasterAli
 	}
 	return nil
 }
+
+func slaveWasRunning(stopSlaveStatus *replicationdatapb.StopSlaveStatus) bool {
+	if stopSlaveStatus == nil {
+		return false
+	}
+
+	return stopSlaveStatus.Before.SlaveIoRunning || stopSlaveStatus.Before.SlaveSqlRunning
+}
+
+func stoppedSlaveStringPosition(stopSlaveStatus *replicationdatapb.StopSlaveStatus) string {
```

Comment:
This function should be unnecessary if we change the RPC to be more user-friendly: we should guarantee that all the fields (hybrid, before, after) are valid as long as err == nil.

Reply:
That's a fair point. I'll remove it after adjusting the code so `After` gets filled in anytime error == nil.

Reply:
Removed.